w1/index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css" />
+</head>
+<body>
+    <header>
+        <!-- 네비게이션 바 -->
+        <div id="head"> 
+            <div class="h_center">
+                <img src="logo.png">
+                <a href="/signin">로그인</a>
+            </div>
+        </div>
+    </header>
+    <main>
+        <!-- c1 -->
+        <div class="c1">
+            <div class="c1_center ">
+                <h1><span>세상의 모든 정보</span>를<br />쉽게 저장하고 관리해 보세요</h1>
+                <a href="/signup">링크 추가하기</a>
+                <img src="w1_img/c1.png">
+            </div>
+        </div>
+        <!-- c2 -->
+        <div class="cont">
+            <div class="contCent">
+                <div class="contTextBox">
+                    <div>
+                        <p class="cont_text1">
+                            <span class="c2_text1">원하는 링크</span>를<br /> 저장하세요
+                        </p>
+                        <p class="cont_text2">
+                            나중에 읽고 싶은 글, 다시보고 싶은 영상,<br /> 사고 싶은 옷, 기억하고 싶은 모든 것을<br /> 한 공간에 저장하세요.
+                        </p>
+                    </div>
+                </div>
+                <div class="contImgBox c2ImgBox">
+                    <div class="c2_imgPack">
+                        <img class="c2_img " src="w1_img/c2_1.png" />
+                        <div class="upParent">
+                            <img class="c2_img line" src="w1_img/c2_2.png" />
+                            <!-- 아이콘 이미지 -->
+                            <img class="c2_up" src="w1_img/c2_icon1.png">
+                        </div>
+                        <img class="c2_img" src="w1_img/c2_3.png" />
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- c3 -->
+        <div class="cont">
+            <div class="contCent">
+                <div class="contImgBox">
+                    <img class="c3_img" src="w1_img/c3.png" />
+                </div>
+                <div class="contTextBox">
+                    <div>
+                        <p class="cont_text1 c3_margin">
+                            링크를 폴더로<br/><span class="c3_text1">관리</span>하세요
+                        </p>
+                        <p class="cont_text2">
+                            나만의 폴더를 무제한으로 만들고<br /> 다양하게 활용할 수 있습니다.
```

Comment:
혹시 whitespace는 의도해주신걸까요~?

```suggestion
                            나만의 폴더를 무제한으로 만들고<br />다양하게 활용할 수 있습니다.
```